webplugin/js/app/kommunicate-ui.js

```diff
@@ -1522,22 +1522,23 @@ KommunicateUI = {
                     'click',
                     this.captureGreetingMessageClick
                 );
-            if (
-                kommunicateIframe.classList.contains(
-                    'chat-popup-widget-vertical'
-                )
-            ) {
-                var heightOfPopup =
-                    document.getElementById('chat-popup-widget-container')
-                        .offsetHeight + 15;
-                var css =
-                    'iframe#kommunicate-widget-iframe.chat-popup-widget-vertical { height:' +
-                    heightOfPopup +
-                    'px!important;}';
-                parent.document.querySelector(
-                    'style#kommunicate-style-sheet'
-                ).innerText += css;
-            }
+            // if (
+            //     kommunicateIframe.classList.contains(
+            //         'chat-popup-widget-vertical'
+            //     )
+            // ) {
+                // var heightOfPopup =
+                //     document.getElementById('chat-popup-widget-container')
+                //         .offsetHeight + 15;
+                     
+                // var css =
+                //     'iframe#kommunicate-widget-iframe.chat-popup-widget-vertical { height:' +
+                //     heightOfPopup +
+                //     'px!important;}';
+                // parent.document.querySelector(
+                //     'style#kommunicate-style-sheet'
+                // ).innerText += css;
+            // }
```

Comment:
if not required then you can remove this

Reply:
done